frontends/web/src/api/devices.ts

```diff
@@ -0,0 +1,282 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { apiGet, apiPost } from '../utils/request';
+
+type TProuctName = 'bitbox' | 'bitbox02' | 'bitbox02-bootloader';
+
+export type TStatus = 'connected'
+    | 'unpaired' | 'pairingFailed'
+    | 'uninitialized' | 'seeded' | 'initialized'
+    | 'require_firmware_upgrade' | 'require_app_upgrade';
+
+export type TStatusBB01 = 'unpaired' | 'pairingFailed'
```

Comment:
can you make a bitbox02.ts and a bitbox01.ts? it would be a lot clearer imho.

Reply:
I like, but there is shared functiontions and types that are not specific to bitbox01 or bitbox02, should we keep the shared ones in `api/devices.ts` or have it in `api/bitbox02.ts`?

Reply:
concept ack, nice!

Reply:
ok updated and since `api/bitbox01.ts` and `api/bitbox02.ts` isn't used it is not anymore part of this PR